src/List.js

```diff
@@ -1,31 +1,53 @@
-import React from 'react';
-import { FirestoreCollection } from 'react-firestore';
+import React, { useContext } from 'react';
+import firebase from 'firebase';
 import NavLinks from './NavLinks';
-import useTokenHook from './useTokenHook';
+import { ItemsContext } from './ItemsContext';
 
 const List = () => {
-  const { token } = useTokenHook();
+  const items = useContext(ItemsContext);
+
+  const now = new Date(Date.now());
```

Comment:
Nice! As far as I know, `new Date()` returns by default the current date and time. Unless there's a special reason for using `Date.now()` that I'm not aware of, it don't think it's necessary to provide an argument to `new Date()`.